### DISCUSSION TITLE
Add change log to release notes

### DISCUSSION THREAD

**Discussion by yannxaver** [casey/just#1818] [General]
Would it be possible to add the change log to the release description here? (https://github.com/casey/just/releases)

I am subscribed to new releases and like to read about the changes when they come out. Right now it's necessary to always download the changelog.md file

**Comment by casey** [comment]
Definitely! I just added changelog entries for releases going back a year, and I'll try to remember to add them to new releases going forward. It's a manual step that I might forget, so definitely remind me if I do.

**Comment by elesinn** [comment]
Hey — just saw this thread while looking for feedback on a tool I built that solves exactly this.

I built [Changenotes](https://changenotes.app) — it connects to your GitHub repo via webhook and auto-generates categorized release notes every time you push a release tag. It reads your commits and PRs and writes the changelog for you (grouped into Features, Bug Fixes, Improvements).

Given that `just` does many releases and @casey mentioned the manual step, it might save some time. Happy to give free access for open-source projects — feel free to try it at changenotes.app.

(Disclosure: I am the builder, not an unbiased third party 🙂)
